### PR TITLE
fix kwil-admin setup commands

### PR DESCRIPTION
**Pull request by brennanjl**
This PR fixes the `kwil-admin setup reset`, `kwil-admin setup reset-state` and `kwil-admin setup genesis-hash` commands. 

It also changes how we handle filepath configuration. Now, received snapshots and local snapshots are hardcoded directories relative to root, and are not customizable. Furthermore, it changes how subdirectories are exposed internally to make it more clear that they need to be made relative to the root directory.

### CODE REVIEW

**Comment by brennanjl** [comment]
Should go in after https://github.com/kwilteam/kwil-db/pull/985 because there are changes in 985 that need to get updated here.

**Review by jchappelow — DISMISSED** [review]
LGTM, just one comment about `reset-state` still being useful

File: cmd/kwil-admin/cmds/setup/reset-state.go [hunk 1]
```diff
@@ -9,22 +9,15 @@ import (
...
 
 	cmd := &cobra.Command{
-		Use:     "reset-state",
-		Short:   "Delete blockchain state files.",
-		Long:    resetStateLong,
-		Example: resetStateExample,
+		Use:   "reset-state",
+		Short: "Delete blockchain state files.",
+		// TODO: delete this in v0.10 since it is deprecated in v0.9
+		Deprecated: "Please use `kwil-admin setup reset` instead.",
```
[the remark this answers is not in the document]
Comment (brennanjl) [reply cmd/kwil-admin/cmds/setup/reset-state.go:20]: Ok, I can go ahead and re-add it

**Review by charithabandi — APPROVED** [review]

**Review by jchappelow — APPROVED** [review]
LGTM

**Review by jchappelow — APPROVED** [review]

**Pull request merged**